libs/packages/sam-formly/src/lib/formly/types/textarea.ts

```diff
@@ -23,6 +24,13 @@ import { FieldType } from '@ngx-formly/core';
         </span>
       </span>
     </div>
+    <span *ngIf="to.tooltipText" class="sds-stack padding-top-105 margin-left-1"
+        [sdsPopover]="to.tooltipText" [sdsPopoverTitle]="to.tooltipTitle"
+        [position]="to.tooltipPosition ? to.tooltipPosition :'right'" tabindex="0"
+        aria-label="info tooltip">
+        <usa-icon [size]="'lg'" [icon]="'info-circle-fill'"></usa-icon>
+      </span>
+    </div>
```

Comment:
Did we want the tooltip besides input or besides label? It might be a bit strange to have tooltip next to something like a text area / input / autocomplete input vs next to the label. Could you add some demo to also see what this looks like? Might need design input